adafruit_platformdetect/board.py

```diff
@@ -409,6 +419,10 @@ def any_udoo_board(self):
         """Check to see if the current board is an UDOO board"""
         return self.id in boards._UDOO_BOARD_IDS
 
+    def any_asus_tinker_board(self):
+        """Check to see if the current board is an ASUS Tinker Board"""
+        return self.ASUS_Tinker_Board
```

Comment:
This function should return a boolean value, so maybe change it to `return self.id == self.ASUS_Tinker_Board`

Reply:
will do and will test